astropy/coordinates/distances.py

```diff
@@ -45,6 +46,8 @@ class Distance(u.SpecificTypeQuantity):
     distmod : float or `~astropy.units.Quantity`
         The distance modulus for this distance. Note that if ``unit`` is not
         provided, a guess will be made at the unit between AU, pc, kpc, and Mpc.
+    parallax : `~astropy.units.Quantity` or `~astropy.coordinates.Angle`
+        The parallax in angular units, 1/distance.
```

Comment:
"1/distance" is confusing here, because something with units of `1/distance` is not an angular unit.  Maybe just leave it off given that saying "the parallax" is sufficient to describe the situation?